CHANGELOG.md

```diff
@@ -12,14 +12,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   the [Item Assets](https://github.com/stac-extensions/item-assets) extension, as mentioned this specification.
 
 ### Changed
-- n/a
+- correct stac-model version in code and publish matching release on pypi, addresses https://github.com/crim-ca/mlm-extension/issues/13
```

Comment:
Should it be under "fixes" ?
Should dd the code quotes around `stac-model`.

Reply:
ah good catches thanks